addons/mail/views/res_config_views.xml

```diff
@@ -21,6 +21,18 @@
                         <field name="alias_domain" placeholder="mycompany.odoo.com" class="oe_inline"/>
                     </div>
                 </xpath>
+                <xpath expr="//group[@name='authentication']" position='before'>
+                    <group name="activity">
+                        <label for="id" string="Activities"/>
+                        <div>
+                            <div>
+                                <button class="oe_link" name="%(mail_activity_type_action)d" type="action">
+                                    <span>Configure the chatter activities</span>
```

Comment:
Not really necessary to duplicate menu entries. Especially for a technical menu. General settings is fine for me.